### PR TITLE
Updates Options SDK Intro

### PR DESCRIPTION
**Pull request by deeleeramone**
Updates the formatting to be consistent and replaces the obsolete examples. I will circle back to this one, but this brings it to the current status and syntax.

### CODE REVIEW

**Comment by kristinWalker1991** [comment]
> Updates the formatting to be consistent and replaces the obsolete examples. I will circle back to this one, but this brings it to the current status and syntax.

https://github.com/OpenBB-finance/OpenBBTerminal/pull/3709#issue-1475679243

File: website/content/sdk/guides/intros/stocks/options.md [hunk 1]
```diff
@@ -1,82 +1,110 @@
...
-from openbb_terminal.sdk import openbb
-
-import pandas as pd
-import numpy as np
-
-import matplotlib.pyplot as plt
-
-%matplotlib widget
-```
+Please be advised that this module is subject to improvements, the guide here may not reflect its current status.
```
Comment (JerBouma) [inline website/content/sdk/guides/intros/stocks/options.md:5]: Please remove this, we don't have to set expectations for the audience. There is **always** room for improvement, nothing is perfect.

File: website/content/sdk/guides/intros/stocks/options.md [hunk 2]
```diff
@@ -1,82 +1,110 @@
...
+To reduce the amount of characters in a script, try importing the module as shown below.
+
+### Import Statements
 
 ```python
-pcr_spy = openbb.stocks.options.pcr(symbol = 'SPY', start_date = '2012-11-03')
-pcr_spy.rename(columns = {'PCR': 'Put/Call Ratio'})
+from openbb_terminal.sdk import openbb
+import pandas as pd
+opt = openbb.stocks.options
```
Comment (JerBouma) [inline website/content/sdk/guides/intros/stocks/options.md:41]: Nowhere in any guide we mention creating a short code for the module so I'd advise not adding it here. It also makes the rest of the examples confusing if you read over this.

File: website/content/sdk/guides/intros/stocks/options.md [hunk 3]
```diff
@@ -1,82 +1,110 @@
...
-The list of expiration dates is generated with:
+The examples in this section will assume these statements are included at the top of the file.
 
-```python
-openbb.stocks.options.option_expirations('SPY')
-```
+### Unusual Options
 
-Get the raw DataFrame for an options chain from yFinance by using:
+`opt.unu` returns a DataFrame with the current day's unusual options, those having a very high volume/open interest ratio. This function returns a Tuple containing the DataFrame and a string. Unpack it like this:
```
Comment (JerBouma) [inline website/content/sdk/guides/intros/stocks/options.md:48]: Replace opt with `openbb.stocks.options`.

File: website/content/sdk/guides/intros/stocks/options.md [hunk 4]
```diff
@@ -1,82 +1,110 @@
...
 
 ```python
-SPY221111 = openbb.stocks.options.chains_yf(symbol = 'SPY', expiry = '2022-11-11')
+unu_df,unu_ts = opt.unu(limit = 500)
```
Comment (JerBouma) [inline website/content/sdk/guides/intros/stocks/options.md:51]: Replace opt with `openbb.stocks.options`.

File: website/content/sdk/guides/intros/stocks/options.md [hunk 5]
```diff
@@ -1,82 +1,110 @@
...
-```python
-from openbb_terminal.sdk import openbb
-import pandas as pd
-import numpy as np
+Get up to ten years of historical Put-Call Ratios.
 
-expirations = openbb.stocks.options.option_expirations(symbol = 'SPY')
-options_df: pd.DataFrame = []
+```python
+opt.pcr(start_date = '2012-01-01', window = 10, symbol = 'SPY')
```
Comment (JerBouma) [inline website/content/sdk/guides/intros/stocks/options.md:75]: Replace opt with `openbb.stocks.options`.

File: website/content/sdk/guides/intros/stocks/options.md [hunk 6]
```diff
@@ -1,82 +1,110 @@
...
 
-options_df = pd.concat(options_df)
+### Hist_CE
 
-options_df.set_index(keys = 'symbol', inplace = True)
+The historical daily data of an individual option, from ChartExchange, includes open interest.
 
-options_df
+```python
+opt.hist_ce(symbol = 'SPY', price = '400', date = '2023-01-20', call = False)
```
Comment (JerBouma) [inline website/content/sdk/guides/intros/stocks/options.md:96]: Replace opt with `openbb.stocks.options`.

File: website/content/sdk/guides/intros/stocks/options.md [hunk 7]
```diff
@@ -1,82 +1,110 @@
...
+|  0 | 2022-12-01 |   8.5  |  10.01 |  7.99 |    8.68 | -0.01251 |    11443 |           47690 |        0       |
+|  1 | 2022-11-30 |  14.44 |  15.68 |  8.79 |    8.79 | -0.3866  |    19763 |           46218 |       -0.01251 |
+|  2 | 2022-11-29 |  14.74 |  15.37 | 14.25 |   14.33 |  0.01415 |      666 |           45105 |       -0.39428 |
+|  3 | 2022-11-28 |  12.49 |  14.86 | 11.94 |   14.13 |  0.27297 |    10643 |           45828 |       -0.3857  |
+| 366 | 2021-05-21 |  37.72 |  37.72 | 37.59 |   37.59 | -0.01079 |        5 |               0 |       -0.76909 |
+| 367 | 2021-05-20 |  38    |  38    | 38    |   38    | -0.07317 |        2 |               0 |       -0.77158 |
+| 368 | 2021-05-19 |  41.67 |  41.67 | 41    |   41    |  0.07471 |       26 |               0 |       -0.78829 |
+| 369 | 2021-05-17 |  37.43 |  38.15 | 37.43 |   38.15 | -0.0453  |        3 |               0 |       -0.77248 |
+
+Stay tuned, more to come soon!
```
Comment (JerBouma) [inline website/content/sdk/guides/intros/stocks/options.md:110]: Please remove this, we don't have to promise things.

**Review by JerBouma — CHANGES_REQUESTED** [review]

File: website/content/sdk/guides/intros/stocks/options.md (hunk 3 above)
Comment (deeleeramone) [reply website/content/sdk/guides/intros/stocks/options.md:48]: done

File: website/content/sdk/guides/intros/stocks/options.md (hunk 2 above)
Comment (deeleeramone) [reply website/content/sdk/guides/intros/stocks/options.md:41]: done

File: website/content/sdk/guides/intros/stocks/options.md (hunk 5 above)
Comment (deeleeramone) [reply website/content/sdk/guides/intros/stocks/options.md:75]: done

File: website/content/sdk/guides/intros/stocks/options.md (hunk 4 above)
Comment (deeleeramone) [reply website/content/sdk/guides/intros/stocks/options.md:51]: done

File: website/content/sdk/guides/intros/stocks/options.md (hunk 7 above)
Comment (deeleeramone) [reply website/content/sdk/guides/intros/stocks/options.md:110]: done

File: website/content/sdk/guides/intros/stocks/options.md (hunk 6 above)
Comment (deeleeramone) [reply website/content/sdk/guides/intros/stocks/options.md:96]: done

**Pull request merged**